assets/model_monitoring/components/tests/unit/test_feature_selector_all.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
do you want to add test case for:
empty_df for input1 and none for input2
empty_df for input1 and empty_df for input2
empty_df for input2
empty_df for input1
None for input1 (this will throw null pointer exception)

Reply:
inputdf1 is null checked in run.py earlier. input_df2 is allowed to be null